spec/models/job_spec.rb

```diff
@@ -274,6 +274,44 @@
     end
   end
 
+  context "job belongs_to task" do
+    before(:each) do
+      @job = Job.create_job("VmScan", :name => "Hello, World!")
+    end
+
+    describe ".create_job" do
+      it "creates job and corresponding task with the same name" do
+        expect(@job.miq_task_id).to eq MiqTask.first.id
```

Comment:
Should set a lookup on name or something more concrete than relying on row order (`first`)